x-pack/plugins/security_solution/public/management/cypress/cypress.config.ts

```diff
@@ -50,6 +54,9 @@ export default defineCypressConfig({
     specPattern: 'public/management/cypress/e2e/mocked_data/',
     experimentalRunAllSpecs: true,
     setupNodeEvents: (on, config) => {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      require('@cypress/grep/src/plugin')(config);
```

Comment:
you not able to use `import` of this at the top of the file?

Reply:
nope, `plugin` doesn't export any module. also, based on the [docs](https://www.npmjs.com/package/@cypress/grep#config-file) this is the usage